### ISSUE TITLE
Gridproxy unavailable at CernVM machines

### ISSUE THREAD

**Issue by laurentdufour** [ganga-devs/ganga#745]
As reported in the lhcb-distributed-analysis mailng list ("problem with ganga on CernVM") Ganga versions v601r22 and above break compatibility with CernVM machines:

```
Ganga In [1]: gridProxy
ERROR    Error: name 'gridProxy' is not defined
ERROR    Unknown/Unexpected ERROR!!
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
/cvmfs/lhcb.cern.ch/lib/lhcb/GANGA/GANGA_v601r24/install/ganga/python/Ganga/GPI/__init__.py in <module>()
----> 1 gridProxy

NameError: name 'gridProxy' is not defined
```

**Comment by rob-c** [comment]
Thanks for reporting this, I've been aware of this for a while we'll look into it depending on manpower.

**Comment by mesmith75** [comment]
Can I clarify - this is just a CernVM issue not CVMFS?

**Comment by laurentdufour** [comment]
According to the mail exchange, it seems to particularly pop up with CernVM. All I know for sure is that I myself experience the issue itself on the Nikhef cluster, which uses CVMfs but not AFS. These are SLC6 machines.

**Comment by mesmith75** [comment]
I managed to reproduce this on the Imperial linux machines. I also had the error at startup
```
ERROR    unknown options [LHCb]ignore_version_check
```

**Comment by rob-c** [comment]
A workaround has been found for this:
Set: `[LCG] GLITE_SETUP=/home` as per #753 in your .gangarc

This should get this working again and give you a gridProxy instance exported to the GPI.
A more permanent solution is being sought via updating the CVMFS site config and this will be fixed in Ganga 6.3.0 properly with a rewrite of the proxy system.

Can someone who has been effected by this report if this workaround fixes your issue so I can close the issue?

Thanks,

Rob

**Comment by mesmith75** [comment]
@rob-c Do you remember the reason for the error:
```
ERROR    unknown options [LHCb]ignore_version_check
```
I have already forgotten it.

**Comment by mattrudolph** [comment]
I was able to get a grid proxy in 6.1.22 inside CernVM using the workaround.  So I assume everything would work afterwards although I haven't tried yet.

**Comment by laurentdufour** [comment]
The workaround seems to work indeed.
Although I still get the LHCb version error.

**Comment by rob-c** [comment]
@laurentdufour Thankfully this error is annoying but harmless. It's due to the site config file pulled in via ganga on cvmfs being very out of date now.
I don't know what the process would be for changing the site config on cvmfs. @po10 do you know what's involved in changing this? If it's just a case of asking the appropriate person to upload a file to the correct place on cvmfs I'll come up with a more up to date config file and this should fix the situation for all users going from 6.1.20 forwards.

**Comment by mesmith75** [comment]
This issue should be resolved now the config has been updated. See the discussion at #753
